qcodes/instrument_drivers/Keysight/keysight_b220x.py

```diff
@@ -0,0 +1,181 @@
+from qcodes import VisaInstrument
+from qcodes.utils.validators import MultiType, Ints, Enum, Lists
+
```

Comment:
Notebook:
> As explained above in Single Route Mode the switch matrix implicitly opens a previous connection when a new connections is made.

Is it a typo? should it be "... the switch matrix implicitly __closes__ a previous connection ..."?

Reply:
Not a typo, but I can rephrase that to make it clearer what is happening. What I mean is that the previous connection is disconnected.

Reply:
ah, not i get it. But it would still be great if you write this more explicitly.

Reply:
Yes, I will rephrase that.

Reply:
Changed it to 
`As explained above in Single Route Mode the switch matrix implicitly disconnects the previous path when a new connections is made.`